x-pack/plugins/apm/public/components/app/RumDashboard/UXMetrics/index.tsx

```diff
@@ -72,7 +81,37 @@ export function UXMetrics() {
       <EuiFlexGroup justifyContent="spaceBetween">
         <EuiFlexItem grow={1} data-cy={`client-metrics`}>
           <EuiTitle size="xs">
-            <h3>{I18LABELS.coreWebVitals}</h3>
+            <h3>
+              {I18LABELS.coreWebVitals}
+              <EuiPopover
+                isOpen={isPopoverOpen}
+                button={
+                  <EuiButtonIcon
+                    onClick={() => setIsPopoverOpen(true)}
+                    color={'text'}
+                    iconType={'questionInCircle'}
+                  />
+                }
+                closePopover={closePopover}
+              >
+                <div style={{ width: '300px' }}>
+                  <EuiText>
+                    <FormattedMessage
+                      id="xpack.apm.ux.dashboard.webCoreVitals.help"
+                      defaultMessage="Learn more about"
+                    />
+                    <EuiLink
+                      href="https://web.dev/vitals/"
+                      external
+                      target="_blank"
+                    >
+                      {' '}
```

Comment:
Wanted to point this out, ignore if it's intentional. I get a lot of these inserted by my editor, so wanted to highlight it for deletion if needed.